packages/pdk-nag/src/packs/aws-prototyping-rules.ts

```diff
@@ -13,25 +13,990 @@ export let PackName = "AwsPrototyping";
+    level: NagMessageLevel.WARN,
+    rule: rules.mediastore.MediaStoreContainerLifecyclePolicy,
+  },
+  {
+    info: "The MediaStore container does not require requests to use SSL.",
+    explanation:
+      "You can use HTTPS (TLS) to help prevent potential attackers from eavesdropping on or manipulating network traffic using person-in-the-middle or similar attacks. You should allow only encrypted connections over HTTPS (TLS) using the aws:SecureTransport condition on MediaStore container policies.",
+    level: NagMessageLevel.WARN,
+    rule: rules.mediastore.MediaStoreContainerSSLRequestsOnly,
+  },
+  {
+    info: "The MSK cluster uses plaintext communication between brokers.",
+    explanation:
+      "TLS communication secures data-in-transit by encrypting the connection between brokers.",
+    level: NagMessageLevel.WARN,
+    rule: rules.msk.MSKBrokerToBrokerTLS,
+  },
+  {
+    info: "The MSK cluster uses plaintext communication between clients and brokers.",
+    explanation:
+      "TLS only communication secures data-in-transit by encrypting the connection between the clients and brokers.",
+    level: NagMessageLevel.WARN,
+    rule: rules.msk.MSKClientToBrokerTLS,
+  },
+  {
+    info: "The Neptune DB instance does have Auto Minor Version Upgrade enabled.",
+    explanation:
+      "The Neptune service regularly releases engine updates. Enabling Auto Minor Version Upgrade will allow the service to automatically apply these upgrades to DB Instances.\n\nExample threat: An actor with a network path to the Neptune cluster or instance can attempt to take advantage of a known vulnerability in a component exposed by Neptune, which may lead to unknown impacts possibly impacting the confidentiality, integrity and availability of the data and resource assets associated with the prototype",
+    level: NagMessageLevel.ERROR,
+    rule: rules.neptune.NeptuneClusterAutomaticMinorVersionUpgrade,
+  },
+  {
+    info: "The Neptune DB cluster does not have encryption at rest enabled.",
+    explanation:
+      "Encrypting data-at-rest protects data confidentiality and prevents unauthorized users from accessing sensitive information.",
+    level: NagMessageLevel.WARN,
+    rule: rules.neptune.NeptuneClusterEncryptionAtRest,
+  },
+  {
+    info: "The Neptune DB cluster does not have IAM Database Authentication enabled.",
+    explanation:
+      "With IAM Database Authentication enabled, the system doesn't have to use a password when connecting to the cluster.",
+    level: NagMessageLevel.WARN,
+    rule: rules.neptune.NeptuneClusterIAMAuth,
+  },
+  {
+    info: "The OpenSearch Service domain does not only grant access via allowlisted IP addresses.",
+    explanation:
+      "Using allowlisted IP addresses helps protect the domain against unauthorized access.",
+    level: NagMessageLevel.WARN,
+    rule: rules.opensearch.OpenSearchAllowlistedIPs,
+  },
+  {
+    info: "The OpenSearch Service domain does not use dedicated master nodes.",
```

Comment:
refer to below comment re: inclusive language.

Reply:
This is the documented technical term - Example https://aws.amazon.com/opensearch-service/faqs/